### DISCUSSION TITLE
How to display variable in content Elsa Designer 3.1

### DISCUSSION THREAD

**Discussion by john-decena** [elsa-workflows/elsa-core#5855] [Q&A] [answered]
<img width="695" alt="image" src="https://github.com/user-attachments/assets/b9d46ea1-de97-4b30-9528-7095b0722351">

There is no intellisense in calling my variable from httpendpoint and display values in httpresponse content. I cannot find the documentation of the syntax or calling the variable

PS. I am new in Elsa

**Comment by Thuylinh28** [comment, marked as answer]
choose type is javaScript in content and use syntax : get + nameVariable() : ex : getPayload()
![image](https://github.com/user-attachments/assets/33d5db1a-d380-45e3-8cae-b9a6338a754f)
